components/kyverno/development/kustomization.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: konflux-kyverno
+
+commonAnnotations:
+  argocd.argoproj.io/sync-wave: "-1"
```

Comment:
why set the sync wave to -1?  AIUI kyverno isn't particularly special in our config.

Reply:
fixed in 304bf2a4e87052acbfd687dd8c07c8fe22646a58

Reply:
I don't see this commit in your branch - can you make sure it's pushed?

Reply:
5e024825aef2e5dccf733390de64b5ff66be9076 wrong branch 🤦🏾